packages/focusvisible/src/index.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 /* Hooks */
-export { useFocusVisible } from './useFocusVisible';
+export { useFocusVisible, IUseFocusVisibleProps, INPUT_TYPES_WHITE_LIST } from './useFocusVisible';
```

Comment:
Why are we exporting `INPUT_TYPES_WHITE_LIST`? This is only used internally to validate input types

Reply:
Yeah I figured it might be but wasn't sure if there were external use cases I didn't know about. I knew you would tell me otherwise 😉. Removing.